argilla-server/src/argilla_server/validators/records.py

```diff
@@ -40,6 +40,7 @@
     "image/svg",
     "image/webp",
 ]
+CHAT_FIELD_MAX_LENGTH = 500
```

Comment:
should we add these to general constants.py?

Reply:
Personally, I find local constants easier to read and maintain.

Reply:
Me too normally but otherwise it ends up being fragmented, right?